lib/workers/repository/process/write.ts

```diff
@@ -35,7 +42,17 @@ export async function writeUpdates(
   for (const branch of branches) {
     addMeta({ branch: branch.branchName });
     const branchExisted = branchExists(branch.branchName);
-    const res = await processBranch(branch);
+    const branchCache =
+      cachedBranches?.find((br) => br.branchName === branch.branchName) ??
+      ({} as BranchCache);
+
+    const managersHash = hasha(
```

Comment:
Shall we use fingerprint terminology here too?

Reply:
> How does this handle the case where no cache is present?


If there's no cache then we fail and store the cache and use it in the next run.

Reply:
So `branchCache` will be an empty object and `branchCache.branchFingerprint` will be `undefined`?